app/services/arcgis_api/mock/fixtures.rb

```diff
@@ -28,6 +28,38 @@ def self.request_candidates_error
       def self.load_response_fixture(filename)
         Rails.root.join('spec', 'fixtures', 'arcgis_responses', filename).read
       end
+
+      private_class_method def self.generate_suggestions(count = 5)
+        {
+          suggestions: Array.new(count) do |index|
```

Comment:
Maybe more conventional / succinct to ... ?

```suggestion
          suggestions: count.times do |index|
```